lib/implore.es6.js

```diff
@@ -60,8 +61,22 @@ export default function implore(request) {
 			'implore requires parameter `method` to be a string'
 		);
 
+		const {requestHeaders={}} = request;
+		invariant(
+			typeof requestHeaders === 'object',
+			'implore requires parameter `requestHeaders` to be an object'
+		);
+
```

Comment:
I mostly agree, especially since `setRequestHeaders` itself does not accept `null`. My only concern here is that XHR itself doesn't require use of `setRequestHeaders`, but we *are* (sort of) requiring that the value be set, given our invariant message here and the docs. It's a fairly typical practice to use `null` as a value indicating something like "all right, I have indeed set this value and am explicitly telling you that I don't want to use it," so I think someone might reasonably want to use it as the value of `request.requestHeaders` here (and expect us to acknowledge their desire not to use it). Instead, we're requiring them to pass an empty object or just leave it undefined, which doesn't seem to allow expressing the intent as well. 

It's a fairly minor issue either way, though, so after leaving this comment I'll nevertheless accept the PR. :-P